test/e2e/specs/editor/blocks/spacer.spec.js

```diff
@@ -43,7 +43,9 @@ test.describe( 'Spacer', () => {
 		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
 
 		await expect(
-			editor.canvas.locator( 'role=document[name="Block: Spacer"i]' )
+			editor.canvas.locator(
+				'role=document[name="Block: Spacer"i] >> css=.components-resizable-box__handle >> [tabindex]'
```

Comment:
This is causing a small visual regression. When you resize the spacer, it kind of loses its nice block outline at the moment.

Reply:
It's important because otherwise we can't differentiate between dragging a block or dragging something else in the block. Block drag (rightly, I think) assumes that if something else within the block is draggable, it also focussed instead of the entire block being focussed, and block drag will ignore it.

Reply:
Can we like check that the currently dragged element is the block wrapper or something instead?

Reply:
The resize handle is in the block wrapper. What can we check?

Reply:
I mean if the element that triggered the drag event is the block wrapper itself.

Reply:
I'm not sure what you mean. But we can't just limit it the block wrapper, otherwise you can drag for example from the image element.

Reply:
> When you resize the spacer, it kind of loses its nice block outline at the moment.

Maybe we can use the block focus outline whenever you focus any element within the block that is not an inner blocks. But idk, what you say here is not really a visual regression imo but rather a fix. For any other block, if you move focus to a button within or some other input field, it removes the outline...

Reply:
Yes, but the issue is that before this PR the focus stayed on the wrapper which gave it a really nice touch.
Maybe, we can have a special case for the resizable blocks (which could be most blocks in the end), when the resizable handles are focused, the block outlines should be visible as well.